_data/versions.yml

```diff
@@ -1350,30 +1350,44 @@ v3.0:
 - title: v3.0.0-alpha1-rc1
   note: |
     24 October 2017
+    
+    This is a pre-release of v3.0.0-alpha1. This release is intended for testing purposes only and is NOT to be used on production systems.
 
-    **What's new**
+    #### What's new
 
     - Support for the [etcd version 3 datastore](https://coreos.com/blog/etcd3-a-new-etcd.html).
 
     - Two new `calicoctl` resources: [BGP Configuration](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/bgpconfig) and [Felix Configuration](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/felixconfig).
 
     - The `calicoctl` Policy resource has been split into [Network Policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy) and [Global Network Policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/globalnetworkpolicy).
+    
+    - The `get`, `apply`, `create`, `delete`, and `replace` commands of `calicoctl` now include an optional `--namespace=<NS>` flag. Refer to the `calicoctl` [Command reference](https://docs.projectcalico.org/v3.0/reference/calicoctl/commands/) section for more details.
+    
+    - The `get` command of `calicoctl` now includes an optional `--all-namespaces` flag. Refer to the [calicoctl get](https://docs.projectcalico.org/v3.0/reference/calicoctl/commands/get) section for more information.
+    
+    - `calicoctl` now ignores the following flags in `get` commands: `--node=<NODE>`, `--orchestrator=<ORCH>`, `--workload=<WORKLOAD>`, and `--scope=<SCOPE>`. These options are now a part of the individual resources and no longer necessary.
+    
+    - `calicoctl` no longer includes a `config` command. To achieve the equivalent functionality: use the `get` command, edit the resource, and then use the `replace` command.
```

Comment:
I think we need to point to the new page once we have it for the configs. i.e. a page describing how to do a `get`, and `apply/replace` on `felixConfiguration` and `bgpConfiguration` to achieve the same result as we use to with the `calicoctl  config` command. We don't have that page right now, but I think @bcreane is working on it